example_project_app/plugins.py

```diff
@@ -19,6 +19,11 @@
 )
 
 
+def callable_function(project=None, user=None):
```

Comment:
This could be renamed into something more descriptive, like `get_project_setting_default()`.

You should create one for each of the three settings scopes and make them utilize the arguments given. This would also make it easy to test these in unit tests.

Maybe make the type of these settings a string and return some meaningful string based on the arguments. Like, project title for a `PROJECT` scope setting, user name for `USER` scope setting, both for `PROJECT_USER` scope setting..